src/Nethermind/Nethermind.Consensus/Producers/TxPoolTxSource.cs

```diff
@@ -87,11 +90,28 @@ public IEnumerable<Transaction> GetTransactions(BlockHeader parent, long gasLimi
             {
                 foreach (Transaction blobTx in selectedBlobTxs)
                 {
-                    yield return blobTx;
+                    if (ResolveBlob(blobTx, out Transaction fullBlobTx))
+                    {
+                        yield return fullBlobTx;
+                    }
                 }
             }
 
             if (_logger.IsDebug) _logger.Debug($"Potentially selected {selectedTransactions} out of {checkedTransactions} pending transactions checked.");
+
+            bool ResolveBlob(Transaction blobTx, out Transaction fullBlobTx)
```

Comment:
nitpick: `TryResolveBlob` would better reflect its bool return type. Furthermore, since it doesn't actually resolve anything, `TryLoadFullBlobTx` would be more accurate. As the PR is already merged and this is a minor detail, feel free to ignore.